.env.example

```diff
@@ -1,4 +1,4 @@
 PORT=3001
 DEV_SERVER_PORT=3000
-TOKEN_SECRET=changeMe
+TOKEN_SECRET=`NRA[&^gb'yb635w
```

Comment:
You should only put examples here. Secret information goes in your `.env` file and we keep that private (not on GitHub).